python/lsst/ts/observatory/control/auxtel/atcs.py

```diff
@@ -817,9 +868,20 @@ async def open_m1_cover(self):
             tel_pos = await self.next_telescope_position(timeout=self.fast_timeout)
 
             if tel_pos.elevationCalculatedAngle[-1] < self.tel_el_operate_pneumatics:
+
+                try:
+                    nasmyth = await self.rem.atmcs.tel_mount_Nasmyth_Encoders.aget(
+                        timeout=self.fast_timeout
+                    )
+                except asyncio.TimeoutError:
+                    raise RuntimeError("Cannot determine nasmyth position.")
+
+                nasmyth_angle = np.mean(nasmyth.nasmyth2CalculatedAngle)
```

Comment:
So this is going to cause issues if we ever want to put the instrument on the other port...
there are probably others lurking as well. Beyond the scope of this ticket but maybe we need to note somewhere that LATISS must be installed on port 2?

Reply:
Yeah, I was lazy when I implemented this the first time. I'll check which one is selected and act accordingly. I am already doing this in other places as well.